manifest/default.xml

```diff
@@ -105,7 +105,7 @@ licenses/APL2.txt.
 
   <project name="goutils" path="godeps/src/github.com/couchbase/goutils" remote="couchbase" revision="f98adca8eb365032cab838ef4d99453931afa112"/>
 
-  <project name="go-blip" path="godeps/src/github.com/couchbase/go-blip" remote="couchbase" revision="3c4edeb16c47cdb3bcd1eeab3b3c5c87832d4923"/>
+  <project name="go-blip" path="godeps/src/github.com/couchbase/go-blip" remote="couchbase" revision="746a6c2c23026766d1a3ace9cca85c5a01f11408"/>
```

Comment:
does go-blip need updating like this in 3.0.xml master ?

https://github.com/couchbase/sync_gateway/pull/6055#pullrequestreview-1276792787